notebooks/rag_eval_ragas.ipynb

```diff
@@ -0,0 +1,5272 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            "strictness": 2
</pre><p>What's strictness?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Updated the metric description with the definition</p>